### PR TITLE
#373: kill divider (closes #373)

### LINKED ISSUE #373: kill divider

### PR DESCRIPTION
**Pull request by francescogior**
Issue #373

# DO NOT MERGE (YET)
- `Divider` was used by `DrodpownMenu/Menu` and `PanelMenu` components
- Both should be probably killed too

### CODE REVIEW

**Comment by francescogior** [comment]
@gabro

**Comment by FrancescoCioria** [comment]
ping @francescogior @gabro

File: src/Divider/divider.scss [hunk 1]
```diff
@@ -1,32 +0,0 @@
...
-  background-color: $pearl;
-  color: $pearl;
-  &.small {
-    margin: 5px;
-  }
-  &.medium {
-    margin: 10px;
-  }
-  &.large {
-    margin: 20px;
```
Comment (gabro) [inline src/Divider/divider.scss]: I liked this margin

**Comment by FrancescoCioria** [comment]
This is PR has been here for ages, I'm tired of it (and it takes 1 minute to re-create a new one if we want, when we want to)

**Pull request closed without merging**